lib/helpers.js

```diff
@@ -88,8 +93,8 @@ const sequence = function (config, cb) {
   return seq;
 };
 
-sequence.run = function (seq) {
-  return require('run-sequence').apply(this, seq);
+sequence.run = function sequenceRun(seq) {
```

Comment:
Because I need the clean `this` for the `.apply`. `this` for an arrow function is is the whole Node module, so it's a bit unwieldily and may break stuff